views/default/tab_client_reissue_cert.pdt

```diff
@@ -105,6 +105,24 @@ if($use_central_api != true )
                         </p>
                     <?php
                     }
+					else if($auth_file_name == 'fileauth.txt'){

```

Comment:
This code is indented to far and the else statements should begin on the same line as the preceding closing brace
`} elseif (...) {`
.
.
.
`} else {`